recipes/re2/bld.bat

```diff
@@ -0,0 +1,17 @@
+mkdir -p build
```

Comment:
In appveyor this gives an error, but doesn't fail the build. It's better to use another name here.